TileStache/Goodies/VecTiles/transform.py

```diff
@@ -685,6 +685,32 @@ def overlap(feature_layers, base_layer, cutting_layer,
         target_attribute, cutting_attrs, keep_geom_type)
 
 
+# map from old or deprecated kind value to the value that we want
+# it to be.
+_deprecated_landuse_kinds = {
+    'station': 'substation',
```

Comment:
Yup, that's what I was trying to say in https://github.com/mapzen/vector-datasource/issues/186#issuecomment-139584028. Do you think it's important to address now? It'll mean pulling the full value of those columns/tags through into Python rather that coalescing them in SQL.